src/ui_common.c

```diff
@@ -106,21 +106,17 @@ unsigned int io_seproxyhal_touch_tx_ok(void) {
                   sizeof(tmpCtx.transactionContext.hash), signature, &sig_len, &info));
     explicit_bzero(&privateKey, sizeof(privateKey));
     // Parity is present in the sequence tag in the legacy API
-    if (tmpContent.txContent.vLength == 0) {
-      // Legacy API
-      G_io_apdu_buffer[0] = 27;
-    }
-    else {
-      // New API
-      // Note that this is wrong for a large v, but the client can always recover
-      G_io_apdu_buffer[0] = (v * 2) + 35;
```

Comment:
The v constant is not used anymore so you need to delete line 93. "uint32_t v = getV(&tmpContent.txContent);"